consumer/nf_mangement.go

```diff
@@ -174,18 +198,32 @@ var SendUpdateNFInstance = func(patchItem []models.PatchItem) (nfProfile models.
 	return
 }
 
-func SendCreateSubscription(nrfUri string, nrfSubscriptionData models.NrfSubscriptionData) (nrfSubData models.NrfSubscriptionData, problemDetails *models.ProblemDetails, err error) {
+func SendCreateSubscription(ctx context.Context, nrfUri string, nrfSubscriptionData models.NrfSubscriptionData) (models.NrfSubscriptionData, *models.ProblemDetails, error) {
 	logger.ConsumerLog.Debugf("Send Create Subscription")
 
+	ctx, span := tracer.Start(ctx, "HTTP POST nrf/subscriptions")
+	defer span.End()
+
+	span.SetAttributes(
+		attribute.String("http.method", "POST"),
+		attribute.String("nf.target", "nrf"),
+		attribute.String("net.peer.name", nrfUri),
+		attribute.String("amf.nf.id", amf_context.AMF_Self().NfId),
+	)
+
 	// Set client and set url
 	configuration := Nnrf_NFManagement.NewConfiguration()
 	configuration.SetBasePath(nrfUri)
 	client := Nnrf_NFManagement.NewAPIClient(configuration)
 
+	var problemDetails *models.ProblemDetails
+	var err error
+	var nrfSubData models.NrfSubscriptionData
```

Comment:
Same as my previous comments
```suggestion
func SendCreateSubscription(ctx context.Context, nrfUri string, nrfSubscriptionData models.NrfSubscriptionData) (nrfSubData models.NrfSubscriptionData, problemDetails *models.ProblemDetails, err error) {
	logger.ConsumerLog.Debugf("Send Create Subscription")

	ctx, span := tracer.Start(ctx, "HTTP POST nrf/subscriptions")
	defer span.End()

	span.SetAttributes(
		attribute.String("http.method", "POST"),
		attribute.String("nf.target", "nrf"),
		attribute.String("net.peer.name", nrfUri),
		attribute.String("amf.nf.id", amf_context.AMF_Self().NfId),
	)

	// Set client and set url
	configuration := Nnrf_NFManagement.NewConfiguration()
	configuration.SetBasePath(nrfUri)
	client := Nnrf_NFManagement.NewAPIClient(configuration)

```

Reply:
Done